pybuild.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+cd /app-src
+apt-get update \
+ && mk-build-deps -i debian/control -t 'apt-get --no-install-recommends -y --force-yes'
+python -m mirbuild.walk -p python-mirbuild package
```

Comment:
These are the same steps in the Dockerfile.

Why don't we change `docker run -e OWNER="$(id -u $USER):$(id -g $USER)" --rm -w /app-src -v $(pwd):/app-src gcr.io/i-lastfm-tools/ubuntu-trusty-buildenv ./pybuild.sh` to instead be a variation of `docker build -f Dockerfile` - it will execute the same instructions, but also make use of the layer cache

Alternatively just use a "plain" ubuntu-trusty-buildenv container?

Reply:
refactored, Dockerfile was removed